File: bench/py/fib.py
# 基准 1：递归斐波那契 fib(26)（与 bench/fib.hn 等价）
def fib(n):
    if n <= 1:
        return n
    return fib(n - 1) + fib(n - 2)

print(fib(26))

File: bench/rs/fib.rs
// 基准 1：递归斐波那契 fib(26)（与 bench/fib.hn 等价）
fn fib(n: u64) -> u64 {
    if n <= 1 {
        return n;
    }
    fib(n - 1) + fib(n - 2)
}

fn main() {
    println!("{}", fib(26));
}
